From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: KUnit: add missing MODULE_DESCRIPTION() macros for lib/*_test.ko
Date: Sat, 1 Jun 2024 16:33:23 -0700

make allmodconfig && make W=1 C=1 reports for lib/*_test.ko:
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/atomic64_test.o
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/hashtable_test.o

Add the missing invocations of the MODULE_DESCRIPTION() macro.

Link: https://lkml.kernel.org/r/20240601-md-lib-test2-v1-1-be764b785f17@quicinc.com
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/atomic64_test.c  |    1 +
 lib/hashtable_test.c |    1 +
 2 files changed, 2 insertions(+)

--- a/lib/atomic64_test.c~kunit-add-missing-module_description-macros-for-lib-_testko
+++ a/lib/atomic64_test.c
@@ -273,4 +273,5 @@ static __exit void test_atomics_exit(voi
 module_init(test_atomics_init);
 module_exit(test_atomics_exit);
 
+MODULE_DESCRIPTION("Testsuite for atomic64_t functions");
 MODULE_LICENSE("GPL");
--- a/lib/hashtable_test.c~kunit-add-missing-module_description-macros-for-lib-_testko
+++ a/lib/hashtable_test.c
@@ -314,4 +314,5 @@ static struct kunit_suite hashtable_test
 
 kunit_test_suites(&hashtable_test_module);
 
+MODULE_DESCRIPTION("KUnit test for the Kernel Hashtable structures");
 MODULE_LICENSE("GPL");
_
